src/main/java/data/TaskManager.java

```diff
@@ -0,0 +1,82 @@
+package data;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.*;
+
+import static data.TaskManagerException.checkIfDateHasTasks;
+import static data.TaskManagerException.checkIfDateInCurrentWeek;
+
+
+public class TaskManager {
+    private static final DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern("dd/MM/yyyy");
+    private static final Map<LocalDate, List<String>> tasks = new HashMap<>();
```

Comment:
Wow! I'm impressed with the usage of hashmaps!

Reply:
Such a innovative way of using hashmaps!